ang/test/civicase/activity/card/directives/activity-card.directive.spec.js

```diff
@@ -65,6 +67,7 @@
           });
           $scope.activity = _.sample(activitiesMockData.get());
           $scope.activity.case_id = _.uniqueId();
+          $scope.activity.type = 'Meeting';
```

Comment:
Is not it better to add the type inside mocked data? that way we dont have to manually add these everytime?

Reply:
I was thinking about that, but this property does not come from the API, it comes from https://github.com/compucorp/uk.co.compucorp.civicase/commit/f351a3108124a169cd7b5eca23ddcefd07265539#diff-fefad92659206a185e765d4eb9a51a26R13

Is it ok to add it to the mock data then?

Reply:
good catch, in that case we should not add it to the mock data, lets remove the commits please.